Sources/NIOHTTP1/HTTPUpgradeHandler.swift

```diff
@@ -171,7 +171,7 @@ public class HTTPServerUpgradeHandler: ChannelInboundHandler {
     /// The core of the upgrade handling logic.
     private func handleUpgrade(ctx: ChannelHandlerContext, request: HTTPRequestHead, requestedProtocols: [String]) -> Bool {
         let connectionHeader = Set(request.headers.getCanonicalForm("connection").map { $0.lowercased() })
-        let allHeaderNames = Set(request.headers.map { $0.name.lowercased() })
+        let allHeaderNames: Set<String> = Set(request.headers.map { $0.name.lowercased() })
```

Comment:
Why does this need an explicit type now?

Reply:
It does not... Somehow this happened. Let me revert